core/src/main/resources/db/changelog.xml

```diff
@@ -14,4 +14,27 @@
     <!-- initial dump for new installations -->
     <include file="changelog_base.xml" relativeToChangelogFile="true" />
 
+    <changeSet author="OliverKlette85" context="new" id="20151229220700">
+        <createTable tableName="user">
+            <column name="id" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="email" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="username" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="password" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="OliverKlette85" context="new" id="20151229222000">
+        <addPrimaryKey columnNames="id" constraintName="user_pkey" tableName="user"/>
+    </changeSet>
+
+    <changeSet author="OliverKlette85" context="new" id="20151229222200">
+        <addUniqueConstraint columnNames="username, id, email" constraintName="user_unique" deferrable="false"
```

Comment:
id shouldn't be part of it, because it is generated. if you have the same name and email, but a generated id it will succeed.